src/pages/charter.js

```diff
@@ -0,0 +1,309 @@
+import Layout from "../layouts";
+import React from 'react'
+import { Helmet } from "react-helmet";
+
+const pageMetaTitle = 'The eBPF Foundation Charter'
+const pageMetaDescription = 'eBPF is a revolutionary technology that can run sandboxed programs in the Linux kernel without changing kernel source code or loading a kernel module.'
+
+const CharterPage = ({ location: {pathname} }) => {
+  return <Layout path={pathname}>
+    <div className="page-wrapper page-charter">
+    <Helmet
+        title={pageMetaTitle}
+
+        meta={[
+          {name: "keywords", content: "ebpf, bpf, contribute, getting started, community"},
```

Comment:
These should also be updated, `contribute` and `getting started` in particular don't sound relevant for this page.

Reply:
The keywords are updated, replacing  `contribute` and `getting started` with `charter`.